File: q2/q2minmax.py
def min(arr) :
    v=arr[0]
    for i in range(len(arr)):
        if v>arr[i]:
            v=arr[i]
    print("smallest value in the array is ",v)
def max(arr):
    v=arr[0]
    for i in range(len(arr)):
        if v<arr[i]:
            v=arr[i]
    print("biggest value in the array is ",v)
    
arr=[1,2,4,45,65,-9]
min(arr)
max(arr)

File: q2/q2minmax.java
public class q2minmax {
	static void min(int arr[]) {
		int s = arr[0];
		for (int i=0;i<arr.length;i++) {
			if (s>arr[i]) {
				s=arr[i];
			}
		}
		System.out.println("Smallest value in the array is " + s);
	}
	static void max(int arr[]) {
		int s = arr[0];
		for (int i=0;i<arr.length;i++) {
			if (s<arr[i]) {
				s=arr[i];
			}
		}
		System.out.println("Largest value in the array is " + s);
	}
	public static void main(String[]args) {
		int arr[]= {1,2,4,5,67,8,9,-3};
		 min(arr);
		 max(arr);
	}
}
